app/models/queen.rb

```diff
@@ -1,13 +1,13 @@
 # Queen behavior.
 class Queen < Piece
-  # Capture, collision, check, and checkmate logic are not
+  # Capture, check, and checkmate logic are not
   # implemented yet and thus ignored.
   def valid_move?(x, y)
     moved?(x, y) &&
       (
```

Comment:
Would be better to use Enumerable.any? - http://apidock.com/ruby/Enumerable/any%3F